moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -92,11 +92,10 @@ void robot_model_loader::RobotModelLoader::configure(const Options &opt)
       rdf_loader_.reset(new rdf_loader::RDFLoader(opt.urdf_string_, opt.srdf_string_));
     else
       rdf_loader_.reset(new rdf_loader::RDFLoader(opt.robot_description_));
-  if (rdf_loader_->getURDF())
-  {
-    const boost::shared_ptr<srdf::Model> &srdf = rdf_loader_->getSRDF() ? rdf_loader_->getSRDF() : boost::shared_ptr<srdf::Model>(new srdf::Model());
-    model_.reset(new robot_model::RobotModel(rdf_loader_->getURDF(), srdf));
-  }
+
```

Comment:
As pointed out in https://github.com/ros-planning/moveit/pull/170#discussion_r76366240, there should be a warning message, if the model is empty.